models/projection/powder/mantid_projection_calculator.py

```diff
@@ -1,22 +1,14 @@
-from math import sqrt,cos
-
-from mantid.simpleapi import SofQWNormalisedPolygon,mtd
-
+from mantid.simpleapi import ConvertToMD
 from models.projection.powder.projection_calculator import ProjectionCalculator
 
+MD_SUFFIX = '_MD'
+
 
 class MantidProjectionCalculator(ProjectionCalculator):
     def calculate_projection(self, input_workspace, output_workspace, qbinning, axis1, axis2):
```

Comment:
The I was told i would be given a naming convention for the projections, I will ask about that today. 
it may be needed in the future